demisto_sdk/commands/format/tests/test_formatting_json_test.py

```diff
@@ -1292,3 +1292,46 @@ def test_json_run_format_old_classifier(self, mocker, pack):
         bs = OldClassifierJSONFormat(input=classifier.path, assume_yes=True)
         bs.run_format()
         assert bs.data['fromVersion'] == VERSION_5_5_0
+
+
+def test_not_updating_id_in_old_json_file(repo):
+    """
+    Given
+        - An old json file with non matching name and id.
+    When
+        - Run format on file
+    Then
+        - Ensure that name and id are still not matching
+    """
+    pack = repo.create_pack()
+    json_incident_type = pack.create_incident_type(name="some_name")
+
+    json_object = BaseUpdateJSON(input=json_incident_type.path)
+    json_object.data['name'] = "name"
+    json_object.data['id'] = "not_name"
+    json_object.old_file = json_object.data.copy()
+    json_object.update_id()
+    assert json_object.data['id'] == "not_name"
+    assert json_object.data['name'] == "name"
+
+
+def test_updating_id_in_old_json_file(repo):
```

Comment:
Not sure this is a valid use case, if a user has changed the ID I am not sure we would want to change it for him.

Reply:
This should be negative like the first one no?